com.github.amit9838.weather.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id" : "com.github.amit9838.weather",
```

Comment:
id must be `io.github` as per the documentation:

https://docs.flatpak.org/en/latest/conventions.html#application-ids